.agents/skills/create-release-note-doc-pr/scripts/create_release_note_doc_pr.py

```
@@ -0,0 +1,370 @@
+            )
+
+        if args.dry_run:
+            print("[OK] Dry run: skipping generated content validation.")
+        else:
+            validate_generated_site_file(site_file)
+
+        run(["git", "add", str(site_file)], cwd=repo_root, dry_run=args.dry_run)
+
+        if not args.dry_run:
+            staged = subprocess.run(
+                ["git", "diff", "--cached", "--quiet"],
```

Ruff (0.15.7)

[error] Starting a process with a partial executable path

(S607)